lib/misc.py

```diff
@@ -99,6 +99,8 @@ def gestalt_logging(cli):
 def import_(cli):
    if (not os.path.exists(cli.path)):
       ch.FATAL("can’t copy: not found: %s" % cli.path)
+   if (cli.xattrs or ("CH_XATTRS" in os.environ)):
+      ch.WARNING("--xattrs unsupported by “ch-image import” (see FAQ)")
```

Comment:
Can you use the global `xattrs_save` here? Then you don't have to duplicate the rather confusing logic to set it (which I think is incorrect here).